packages/cache/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@joshdb/cache",
+  "version": "1.0.0",
+  "description": "A cache middleware for Josh",
+  "author": "@joshdb",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "browser": "dist/index.umd.js",
+  "unpkg": "dist/index.umd.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.js",
+    "types": "./dist/index.d.ts"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "test": "jest",
+    "docs": "typedoc-json-parser",
+    "lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
+    "build": "rollup -c rollup.config.ts",
+    "prepack": "rollup-type-bundler",
+    "bump": "cliff-jumper",
+    "check-update": "cliff-jumper --dry-run"
+  },
+  "dependencies": {
+    "@joshdb/middleware": "next",
```

Comment:
This should target the version the other packages use in this workspace.

Reply:
Oh this seems to be what `yarn generate` outputs, perhaps that command should require() the map package.json and grab the versions from there?

Reply:
We could possibly do something like that. Or we could just import it and use `MapProvider.version`

Reply:
That works, maybe open an issue?